storage/boltdb/net-state.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package boltdb
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/boltdb/bolt"
+)
+
+// File Path and Value are saved to boltdb
+type File struct {
+	Path  string `json:"path"`
+	Value string `json:"value"`
+}
+
+const (
+	fileBucketName = "files"
+)
+
+var (
+	errFileNotFound = Error.New("error file not found")
+)
+
+func (client *Client) Put(file File) error {
+	return client.db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte(fileBucketName))
+		if err != nil {
+			return err
+		}
+
+		fileKey := []byte(file.Path)
+
+		fileBytes, err := json.Marshal(file.Value)
+		if err != nil {
+			log.Println(err)
+		}
+
+		return b.Put(fileKey, fileBytes)
+	})
+}
+
+func (client *Client) Get(fileKey []byte) (File, error) {
+	var fileInfo File
+	err := client.db.Update(func(tx *bolt.Tx) error {
+		b := tx.Bucket([]byte(fileBucketName))
+		v := b.Get(fileKey)
+		if v == nil {
+			return errFileNotFound
+		}
+		unmarshalErr := json.Unmarshal(v, &fileInfo.Value)
+		return unmarshalErr
```

Comment:
i prefer just sticking with `err` for error variable names, but you could also do

```
return json.Unmarshal(...
```

the reason for always using `err` or never using it is because with Go you really get into the habit of typing and seeing `if err != nil {`, so you want that to be the right thing. this might be a hard bug to spot:

```
unmarshalErr := json.Unmarshal(v, &fileInfo.Value)
if err != nil {
  return err
}
```

or worse

```
unmarshalErr := json.Unmarshal(v, &fileInfo.Value)
if unmarshalErr != nil {
  return err
}
```